numpy/core/arrayprint.py

```diff
@@ -455,11 +446,27 @@ def array2string(a, max_line_width=None, precision=None,
 
     """
 
+    if max_line_width is None:
+        max_line_width = _line_width
+
+    if precision is None:
+        precision = _float_output_precision
+
+    if suppress_small is None:
+        suppress_small = _float_output_suppress_small
+
+    if formatter is None:
+        formatter = _formatter
+
     if a.shape == ():
         x = a.item()
-        if isinstance(x, tuple):
-            x = _convert_arrays(x)
```

Comment:
I removed all `_convert_arrays` and replace the code on `rank == 0` with `ValueError` to prevent misuse of `_formatArray`.